### ISSUE TITLE
[Bug]: Storybook 9 - A11y addon highlight does not work as expected

### ISSUE THREAD

**Issue by yannbf** [storybookjs/storybook#30968]
### Describe the bug

**Repo:** MealDrop
**Branch:** `sbtest-qa`, commit `db25fba146e`

**Steps:**
- Visit http://localhost:6006/?path=/story/pages-checkoutpage--empty&addonPanel=storybook/a11y/panel&a11ySelection=passes.aria-prohibited-attr.3
- Notice that the element is not highlighted:

<img width="500" alt="Image" src="https://github.com/user-attachments/assets/4463df1b-0391-4058-aa88-3a93ba1c10b0" />

The expected behavior would be to see it highlighted as such:

<img width="500" alt="Image" src="https://github.com/user-attachments/assets/cd7bf990-d9d5-4c31-b6d1-624a4b6a2a5f" />

### Reproduction link

-

### System

```bash
-
```
